packages/report-server/src/__tests__/reportBuilder/testUtils/entityApiMock.ts

```diff
@@ -15,4 +18,38 @@ export const entityApiMock = (entities: Record<string, Record<string, string>[]>
     const { fields } = queryOptions;
     return fields ? foundEntities.map(e => pick(e, fields)) : foundEntities;
   },
+  getDescendantsOfEntities: async (
```

Comment:
Hehe thanks 😄 idk why but queue's are on of my fav things to write, takes me back to my uni days 😂  

I thought about moving it to a more central place so that it can be re-used. I think perhaps implementing a complete `MockTupaiaApi` class would be really good at some point so that when we make changes to any of the microservices we only need to update the mocks in one place.

I'll create a card for it but I think it's out of scope for this ticket? 👍

Reply:
Yeah I was thinking something similar, a card would be good thanks